server/app/views/applicant/ApplicantCommonIntakeUpsellCreateAccountView.java

```diff
@@ -0,0 +1,187 @@
+          redirectButton(
+                  "sign-in",
+                  messages.at(MessageKey.BUTTON_LOGIN.getKeyName()),
+                  controllers.routes.LoginController.applicantLogin(Optional.of(redirectTo)).url())
+              .withClasses(ApplicantStyles.BUTTON_UPSELL_SECONDARY_ACTION),
+          redirectButton(
+                  "sign-up",
+                  messages.at(MessageKey.BUTTON_CREATE_AN_ACCOUNT.getKeyName()),
+                  controllers.routes.LoginController.register().url())
+              .withClasses(ApplicantStyles.BUTTON_UPSELL_PRIMARY_ACTION));
+    } else {
+      actionButtonsBuilder.add(
+          redirectButton(
+                  "apply-to-programs",
+                  messages.at(MessageKey.BUTTON_APPLY_TO_PROGRAMS.getKeyName()),
+                  controllers.applicant.routes.ApplicantProgramsController.index(applicantId).url())
+              .withClasses(ApplicantStyles.BUTTON_UPSELL_PRIMARY_ACTION));
+    }
+
+    var content =
+        div()
+            .withClasses(ApplicantStyles.PROGRAM_INFORMATION_BOX)
+            .with(
+                h1(title).withClasses("text-3xl", "text-black", "font-bold", "mb-4"),
+                eligibleProgramsSection(eligiblePrograms, messages, isTrustedIntermediary)
+                    .withClasses("mb-4"),
+                section()
+                    .condWith(
+                        shouldUpsell,
+                        h2(messages.at(MessageKey.TITLE_CREATE_AN_ACCOUNT.getKeyName()))
+                            .withClasses("mb-4", "font-bold"),
+                        div(messages.at(MessageKey.CONTENT_PLEASE_CREATE_ACCOUNT.getKeyName()))
+                            .withClasses("mb-4"))
+                    .with(
+                        div()
+                            .withClasses(
+                                "flex",
+                                "flex-col",
+                                "gap-4",
+                                "justify-end",
+                                StyleUtils.responsiveMedium("flex-row"))
+                            .with(actionButtonsBuilder.build())));
+
+    bannerMessage.ifPresent(bundle::addToastMessages);
+    bundle.addMainStyles(ApplicantStyles.MAIN_PROGRAM_APPLICATION).addMainContent(content);
+    return layout.renderWithNav(request, applicantName, messages, bundle);
+  }
+
+  private SectionTag eligibleProgramsSection(
+      ImmutableList<ApplicantService.ApplicantProgramData> eligiblePrograms,
+      Messages messages,
+      boolean isTrustedIntermediary) {
+    var eligibleProgramsSection = section();
+
+    if (eligiblePrograms.isEmpty()) {
+      var moreLink =
+          new LinkElement()
+              .setStyles("underline")
+              .setText(
+                  messages.at(
+                      MessageKey.CONTENT_COMMON_INTAKE_NO_MATCHING_PROGRAMS_LINK_TEXT.getKeyName()))
+              .setHref("https://access.arkansas.gov/Learn/Home")
```

Comment:
**issue (blocking):** hard-coded AR link?

Reply:
Added a TODO. The link text in the messages file is also AR specific.

We can add these as env variables or hardcode until we have an admin UI for these sort of customizations, which is the same issue as the login screen @MichaelZetune was running into.

Reply:
Shouldn't we add these as env variables now, given that the link does not need to be translated? We may eventually move many env vars to a UI, but no reason to hardcode in the interim imo.

Reply:
Moved to an env var.